scripts/build_push_proxy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Maybe this script is not necessary at all? What do the reviewers think?

Reply:
nice script! yeah, CI/Hokusai will just build the canonical image (`artsy-unleash`), so using this script to locally build the proxy image, is a solution.